mono/mini/debugger-agent.c

```diff
@@ -1091,12 +1091,18 @@ mono_debugger_get_generate_debug_info ()
 	return disable_optimizations;
 }
 
+// Declare transport_close2 method
+static void transport_close2(void);
+
 MONO_API void
 mono_debugger_disconnect ()
```

Comment:
Is this a new function we added or are we changing existing behaviour here?

Would it make sense to put the transport_close2 call into a new function like? mono_debugger_disconnect_external() ?

Reply:
transport_close2 is an existing function. In this PR I'm just modifying the implementation of mono_debugger_disconnect

Reply:
I'm asking if you originally added mono_debugger_disconnect or if it already existed and this change could be breaking change.

Reply:
mono_debugger_disconnect was added by Jonathan Chambers with the other new APIs needed for scripting/no-cost-editor-attaching

Reply:
Okay, great :-) Then no existing functionality will break.